analyzers/src/ITs.JsonParser/IssueParser.cs

```diff
@@ -58,11 +58,17 @@ private static IEnumerable<OutputReport> ExplodeToRuleIssues(InputReport inputRe
         var allIssues = inputReport.Sarif
             .AllIssues()
             .Where(x => RxSonarRule.IsMatch(x.RuleId))
-            .OrderBy(x => x.Order())
             .GroupBy(x => x.RuleId);
         foreach (var issuesByRule in allIssues)
         {
-            var issues = new RuleIssues { Issues = issuesByRule.Select(x => new RuleIssue(x)).ToArray() };
+            var issues = new RuleIssues
+            {
+                Issues = issuesByRule.Select(x => new RuleIssue(x))
+                    .OrderBy(x => x.Uri)
+                    .ThenBy(x => x?.Location)
```

Comment:
The `null` scenario should be tested

Reply:
I removed as discussed the null conditional operator, a `x` won't be null. 
This should increase the coverage.